installer/install.sh

```diff
@@ -13,11 +16,16 @@ readonly VER_LT=2
 oc_install_dir="/usr/local/bin"
 oc_version_comparison=${VER_LT}
 
-#Minimal version requirements
+#####################################################################
+# Minimal version requirements                                      #
+#####################################################################
 readonly MIN_PYTHON_VERSION=2.7
 readonly MIN_ANSIBLE_VERSION=2.6
 readonly MIN_OCP_CLIENT_TOOL=3.9
 
+#####################################################################
+# Utils functions                                                   #
+#####################################################################
```

Comment:
For example, it probably should go to an utils.sh file and be imported. However, now we can find and know where exactly looking for to find something.